yo1k/tic_tac_toe/kernel/game.py

```diff
@@ -11,15 +11,27 @@ class Mark(Enum):
     O = auto()
 
 
+@eq
+class PlayerID:
+    def __init__(self, idx: int):
+        self.idx: int = idx
+
+    def __repr__(self) -> str:
+        return (f"{type(self).__qualname__}("
+                f"idx={self.idx})")
+
+
 @eq
 class Player:
-    def __init__(self, mark: Mark):
+    def __init__(self, mark: Mark, id_: PlayerID):
```

Comment:
I already have `id` as the first constructor parameter. Let's maintain the same order in Python (this is also relevant for the `_new_player` function in tests).

Reply:
Done.